doc/releases/release-notes-2.1.rst

```diff
@@ -31,7 +31,12 @@ Boards & SoC Support
 Drivers and Sensors
 *******************
 
-* TBD
+* EEPROM
+
+  * Added EEPROM device driver API
+  * Added Atmel AT24 (and compatible) I2C EEPROM driver
+  * Added Atmel AT25 (and compatible) SPI EEPROM driver
+  * Added native_posix EEPROM emulation driver
```

Comment:
We are repeating this one :) ( #20914 ), I guess they can remove it after if need be

Reply:
@aescolar We'll see who wins the race to master. I can remove it from here, if you'd prefer that.

Reply:
There is going to be a merge conflict anyhow for whoever comes second.. so no need to remove yet.